.github/workflows/deb.yml

```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build Debian Package
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ gtk4 ]
+  pull_request:
+    branches: [ gtk4 ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - run: sudo apt install build-essential debhelper meson gettext pkg-config libglib2.0-dev gjs appstream
```

Comment:
In my computer, it does not build without the new build dependencies added [here](https://github.com/johnfactotum/foliate/pull/1123) (they were not necessary in Foliate version 2).

Reply:
Ah, now it does not build because Ubuntu 22.04 repositories have too old versions of gjs (and probably other software too). We probably need to wait until 24.04 GitHub actions for it to generate a deb package.

Reply:
Or add an option to skip the dependency checks. But that makes no sense, either. Those dependencies are supposed to be run-time dependencies. Either we're using Meson wrong and should specify run-time dependencies another way, or Meson should be skipping these checks automatically when building a package.

Reply:
I don't think we're doing anything wrong, cause Meson says:
```
Run-time dependency gjs-1.0 found: YES 1.78.0
Run-time dependency gtk4 found: YES 4.12.3
Run-time dependency libadwaita-1 found: YES 1.4.0
Run-time dependency webkitgtk-6.0 found: YES 2.42.2
```

So they are *run-time* dependencies indeed.

This means it could be the same bug that requires you to install `desktop-file-utils`.